Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -933,6 +1048,114 @@ void UpdatePlanes(std::vector<Frame::Plane::Identifier>& updatedPlanes, std::vec
             }
         }
 
+#if (__IPHONE_OS_VERSION_MAX_ALLOWED >= 130400)
+        /**
+         Updates existing meshes in place, gets the list of updated/created mesh IDs and removed mesh IDs.
+         */
+        void UpdateMeshes(std::vector<Frame::Mesh::Identifier>& updatedMeshes, std::vector<Frame::Mesh::Identifier>& deletedMeshes) {
+            if (![sessionDelegate TrySetMeshDetectorEnabled: true]) {
```

Comment:
Should this be setting the mesh detector to enabled, or just checking if it's enabled? It seems odd that the UpdateMeshes function would directly enable the mesh detector, but I might be missing something